packages/snaps-cli/src/webpack/loaders/function.ts

```diff
@@ -51,12 +56,3 @@ export function getFunctionLoader<Options>(
     },
   };
 }
-
-// When running as CJS, we need to export the loader as a default export, since
```

Comment:
Do we not still want to allow the CLI to run under CJS?

Reply:
This was only needed for `tsup`. `ts-bridge` doesn't have the same issue.

Reply:
Ah 👍